yb-voyager/cmd/exportData.go

```diff
@@ -887,12 +890,18 @@ func applyTableListFlagsOnFullListAndAddLeafPartitions(fullTableList []sqlname.N
 		//this is only for removing  the mid level partitioned table from fullTableList
 		//by passing false in `addAllLeafPartitions` boolean flag which  means this function won't the leaf parittions again it will just filter the list based on type
 		//i.e. only add if its a normal, root, or leaf table.
+		log.Infof("keeping only leaf and root tables in full table list - %v", lo.Map(includeTableList, func(t sqlname.NameTuple, _ int) string {
```

Comment:
nit: printing list of tables here might be unnecessary, since this list of tables and the one logged at the end of "applyFilterAndAddLeafTable" will be the same.

Reply:
This is logging before and after adding leaf, so the list can change in between, right